docs/content/mobile/ios/local-deploy.md

```diff
@@ -39,7 +39,9 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
     * If you choose **Build**, instead of **Build and Run**, Unity generates a XCode project and opens the folder containing the project.
   1. Once the game is deployed and started on the Simulator, you see an empty text field and a **Connect** button: Select **Connect**.
 
-    > You don’t need to enter anything in the text field.
+    > You don’t need to enter anything in the text field,
+
+  1. Once you've installed or ran app on the Simulator you can run **SpatialOS** > **Launch Mobile Client** > **iOS Client** to launch mobile client with IP address field prefilled with a value from **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**.
```

Comment:
this seems to be the wrong place. I would maybe add it as a call-out underneath the last point:

something along:
```
> **Note**:  We provide a setting in the GDK Tools configuration window to fill in your local IP address. In the Unity Editor, go to  **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment**. Once you have built the app via XCode, you can launch the iOS simulator via **SpatialOS** > **Launch Mobile Client** > **iOS Client** in the Unity Editor. This starts your app on your iOS simulator and passes in the IP address that you defined as an argument. 
```

probably best for @ElleEss  to do a check through that text as well